yeat/tests/test_bandage.py

```diff
@@ -15,37 +15,31 @@
 import subprocess
 from unittest.mock import patch
 from yeat.tests import data_file
-from yeat.workflows import bandage
+from yeat.workflows import snakefiles
 
 
 @pytest.mark.bandage
 def test_check_bandage():
-    assert bandage.check_bandage()
+    assert snakefiles.check_bandage()
 
 
 @pytest.mark.bandage
 @patch("subprocess.run")
 def test_bandage_subprocess_failed(function_mock):
     function_mock.return_value = subprocess.CompletedProcess(["Bandage", "--help"], 1)
-    assert bandage.check_bandage() == False
+    with pytest.warns(UserWarning, match=r"Unable to run Bandage; skipping Bandage"):
+        assert snakefiles.check_bandage() == False
 
 
 @pytest.mark.short
-@patch.dict(os.environ, {"PATH": "ROUGE"})
+@patch.dict(os.environ, {"PATH": "DNE"})
```

Comment:
*[confused caveman grunt]*

Reply:
For better readability, I changed the word "ROUGE" to "DNE" which stands for "Does Not Exist".